Source/FullScreenPokemon.ts

```diff
@@ -2497,6 +2509,7 @@ module FullScreenPokemon {
             if (other.dialogOptions) {
                 this.animateCharacterDialogOptions(thing, other, other.dialogOptions);
             } else if (other.trainer && !(<IEnemy>other).alreadyBattled) {
+                this.autoSave();
```

Comment:
I put this line here because I thought I was having autoSave before and after battles
I am going to remove all these calls in different places before a battle starts, and just have one at the very start of startBattle